build/config/common-settings.json

```diff
@@ -257,7 +257,7 @@
    "ums.volume.sdcard1.enabled": true,
    "ums.volume.extsdcard.enabled": true,
    "ums.volume.external.enabled": true,
-   "usb.transfer": "0",
+   "usb.transfer": "auto",
```

Comment:
Right - I was assuming that we didn't want to change whatever setting the user currently has, just the default for new phones.